src/test/java/org/reaktivity/k3po/nukleus/ext/DuplexIT.java

```diff
@@ -318,6 +318,66 @@ public void shouldThrottleInitialOnlyServerSentData() throws Exception
 
     @Test
     @Specification({
+        "server.sent.throttle.stream/client",
+        "server.sent.throttle.stream/server"
+    })
+    public void shouldThrottleClientSentDataPerStream() throws Exception
+    {
+        k3po.finish();
+    }
+
+    // should pass because even if throttling is disabled, the receiver window is big enough to receive the message
+    @Test
+    @Specification({
+        "client.sent.throttle.off/client",
+        "client.sent.throttle.off/server"
```

Comment:
This test will not pass reliably because there is a race between the client sending the first Data frame and the server sending the initial Window frame. If the server receives the Data frame before it writes the Window frame the test will fail because the server will (correctly) reply with a Reset frame. Moreover, there is already a test that shows that receiving a message that fits in the initial window works as expected: client.sent.throttle.initial.only. So please remove this test.

Reply:
Indeed, I haven't thought about the race, the test passed for me always, but I guess I was just lucky. I've removed it.